src/tiling/tiler.rs

```diff
@@ -86,8 +88,15 @@ impl TilingInfo {
     } else {
       0
     };
+    let min_tile_rows_ratelimit_log2 = if min_tiles_ratelimit_log2 > tile_cols_log2 {
+      min_tiles_ratelimit_log2 - tile_cols_log2
```

Comment:
isn't that a `saturating_sub()`?

Reply:
yes but I'd rather have it match the other case right below